src/Aquifer.Jobs/Subscribers/NotificationSubscriber.cs

```diff
@@ -0,0 +1,174 @@
+﻿using System.Data.Common;
+using System.Web;
+using Aquifer.Common.Jobs;
+using Aquifer.Common.Jobs.Messages;
+using Aquifer.Common.Services;
+using Aquifer.Data;
+using Aquifer.Jobs.Configuration;
+using Azure.Storage.Queues.Models;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Aquifer.Jobs.Subscribers;
+
+public sealed class NotificationSubscriber(
+    IOptions<ConfigurationOptions> _configurationOptions,
+    AquiferDbContext _dbContext,
+    IEmailService _emailService,
+    ILogger<NotificationSubscriber> _logger)
+{
+    [Function(nameof(SendProjectStartedNotification))]
+    public async Task SendProjectStartedNotification(
+        [QueueTrigger(Queues.SendProjectStartedNotification)] QueueMessage queueMessage,
+        CancellationToken ct)
+    {
+        var message = queueMessage.Deserialize<SendProjectStartedNotificationMessage, NotificationSubscriber>(_logger);
+
+        var project = await _dbContext.Projects
+            .Include(projectEntity => projectEntity.CompanyLeadUser)
+            .FirstAsync(p => p.Id == message.ProjectId, ct);
+
+        var companyLeadUser = project.CompanyLeadUser;
+        if (companyLeadUser is not null && companyLeadUser.AquiferNotificationsEnabled)
+        {
+            var templatedEmail = new TemplatedEmail(
+                From: NotificationsHelper.NotificationSenderEmailAddress,
+                // Template Designer: https://mc.sendgrid.com/dynamic-templates/d-7760ec3b5ce34b179384d4783cc1bd81/version/e83075a3-ba61-4d42-922f-9fd5df4ee45c/editor
+                TemplateId: "d-7760ec3b5ce34b179384d4783cc1bd81",
```

Comment:
See https://biblionexusworkspace.slack.com/archives/C05K5N7F44Q/p1729881062250939 for design review of this email template.

Reply:
This feels like it should be a config value, not hard-coded magic string.